x-pack/plugins/security_solution/public/common/components/risk_score/risk_score_onboarding/utils.ts

```diff
@@ -0,0 +1,611 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { HttpSetup, NotificationsStart, ThemeServiceStart } from '@kbn/core/public';
+import type { DashboardStart } from '@kbn/dashboard-plugin/public';
+import { RiskScoreEntity } from '../../../../../common/search_strategy';
+import * as utils from '../../../../../common/utils/risk_score_modules';
+import type { inputsModel } from '../../../store';
+
+import {
+  createIngestPipeline,
+  createIndices,
+  createStoredScript,
+  createTransform,
+  startTransforms,
+  deleteStoredScripts,
+  deleteTransforms,
+  deleteIngestPipelines,
+  stopTransforms,
+  bulkCreatePrebuiltSavedObjects,
+  bulkDeletePrebuiltSavedObjects,
+} from '../../../../risk_score/containers/onboarding/api';
+import {
+  INGEST_PIPELINE_DELETION_ERROR_MESSAGE,
+  INSTALLATION_ERROR,
+  START_TRANSFORMS_ERROR_MESSAGE,
+  TRANSFORM_CREATION_ERROR_MESSAGE,
+  TRANSFORM_DELETION_ERROR_MESSAGE,
+  UNINSTALLATION_ERROR,
+} from '../../../../risk_score/containers/onboarding/api/translations';
+
+interface InstallRiskyScoreModule {
+  dashboard?: DashboardStart;
+  http: HttpSetup;
+  notifications?: NotificationsStart;
+  refetch?: inputsModel.Refetch;
+  renderDashboardLink?: (message: string, dashboardUrl: string) => React.ReactNode;
+  renderDocLink?: (message: string) => React.ReactNode;
+  spaceId?: string;
+  theme?: ThemeServiceStart;
+  timerange: {
+    from: string;
+    to: string;
+  };
+}
+
+type UpgradeRiskyScoreModule = InstallRiskyScoreModule;
+
+export const installHostRiskScoreModule = async ({
```

Comment:
this and `installUserRiskScoreModule` may be similar enough to combine as well?

Reply:
Users rick score has one step less than hosts risk score, so the calls' count does not the same between the two tests, so I left this one separate and combined rest of the tests.